fastlane/Fastfile

```diff
@@ -119,13 +117,18 @@ private_lane :validate_repo do |options|
 
   rubocop if File.exist?("../.rubocop.yml") # some project don't yet use rubocop
 
-  # Verifying the --help command
+  # Restricted from running on Travis CI because it causes builds to hang.
   #
-  binary_path = File.join("..", "bin", tool_name)
-  unless no_binary.include? tool_name
-    content = sh "PAGER=cat #{binary_path} --help"
-    ["--version", "https://fastlane.tools", tool_name].each do |current|
-      UI.user_error!("--help missing information: '#{current}'") unless content.include? current
+  unless ENV['TRAVIS']
```

Comment:
Discussed offline, `FastlaneCore::Helper.ci?` would exclude all CIs, but we'd like to allow for testing on alternative CIs to see if they have the same hanging problem.